src/ol/format/kmlformat.js

```diff
@@ -1712,6 +1753,71 @@ ol.format.KML.prototype.readNameFromNode = function(node) {
 
 
 /**
+ * @param {Document|Node|string} source Souce.
+ * @return {Array.<Object>} Network links.
+ * @api stable
```

Comment:
I don't think we need the stability marker at this point. So please leave `@api` but remove `stable`.